src/components/shared/BottomNavigator.jsx

```diff
@@ -26,10 +26,10 @@ const BottomNavWrapper = styled.div`
 
   width: 100%;
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  flex-direction: row;
+  justify-content: flex-end;
   align-items: flex-end;
-  padding: 24px 40px;
+  padding: 24px 48px;
   border-top: 2px solid;
```

Comment:
P2: 색상도 명시적으로 기입해주시면 좋을 것 같아요!

```suggestion
  border-top: 2px solid #000000;
```